pkg/manifests/manifests.go

```diff
@@ -6,10 +6,25 @@ import (
 
 	monv1 "github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1"
 	"k8s.io/apimachinery/pkg/util/yaml"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 )
 
 var (
-	CephRules = "jsonnet/manifests/ceph-prometheus-rules.yaml"
+	manLog = logf.Log.WithName("manifests_manifests")
+	rules  = map[string]map[string]string{
```

Comment:
`rules` has a static set of storages and their versions with corresponding rules.yaml files. Isn't it a good idea to keep a config file with values and read from the file, instead of changing the code?

Reply:
Sounds good. Will work on it.